### PR TITLE
FEATURE: Grab more product details and test

### PR DESCRIPTION
**Pull request by spagi**

### CODE REVIEW

File: run.php [hunk 1]
```diff
@@ -14,7 +15,11 @@
 
 $productFactory = new ProductFactory();
 
-$grabber = new ShopGrabber($productFactory, new Dom());
+$dom =new Dom();
```
Comment (Kapsid) [inline run.php]: Space after =

File: src/Service/ProductDetailGrabber.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+        return $rating->text;
+    }
+
+    public function getLabel(?string $link): ?string
+    {
+        $url = sprintf(self::PRODUCT_DETAIL_URL, $link);
+        $this->dom->loadFromUrl($url);
+
+        $labelDom = $this->dom->find('h1');
```
Comment (Kapsid) [inline src/Service/ProductDetailGrabber.php:42]: Are you sure that there are not more 'h1' ? I know that its a best practise to not add more H1 to the page but I believe that its possible to do it and a lot of websites have those :-D

If you checked and its the only one feel free to resolve this.

**Review by Kapsid — CHANGES_REQUESTED** [review]
Good job! Just two small comments and we can merge.

File: src/Service/ProductDetailGrabber.php (hunk 2 above)
Comment (spagi) [reply src/Service/ProductDetailGrabber.php:42]: Luckily there is only one h1

**Pull request merged**